src/config.rs

```diff
@@ -29,8 +29,8 @@ impl Default for Config {
         Config {
             source: "./".to_owned(),
             dest: "./".to_owned(),
-            layouts: "_layouts".to_owned(),
-            drafts: "_drafts".to_owned(),
+            templates: "templates".to_owned(),
+            drafts: "drafts".to_owned(),
```

Comment:
Reason you are removing the leading "_"?  I believe those tell cobalt to ignore these folders so they don't get processed or copied over as assets?